serverless-configs/aws-cors-configuration.yml

```diff
@@ -5,4 +5,5 @@ headers:
   - Authorization
   - Earthdata-ENV
   - Origin
-  - User-Agent
\ No newline at end of file
+  - User-Agent
+  - Content-Type
```

Comment:
Do we still need this change after switching to a regular 200 response?

Reply:
Yeah, the first time we deployed to sit and tried, it broke due to the same CORS error we were seeing before, due to Content-Type not being exposed.